ml-agents/mlagents/trainers/learn.py

```diff
@@ -30,7 +29,9 @@
 from mlagents_envs.side_channel.engine_configuration_channel import EngineConfig
 from mlagents_envs.exception import UnityEnvironmentException
 from mlagents_envs.timers import hierarchical_timer, get_timer_tree
-from mlagents.logging_util import create_logger
+from mlagents_envs import logging_util
```

Comment:
Do we need to import everything ?

Reply:
I received the opposite feedback from @ervteng on Friday, see https://github.com/Unity-Technologies/ml-agents/pull/3703#discussion_r399547292

I think this is preferable when we want to use the logging levels; it's clearer in the code to see `logging_util.INFO` instead of `INFO`.

Another alternative is to use `import logging` and disable the "banned module" error here.